app/src/main/java/ar/scacchipa/twittercloneapp/presentation/MainCloneActivity.kt

```diff
@@ -1,4 +1,4 @@
-package ar.scacchipa.twittercloneapp.component
+package ar.scacchipa.twittercloneapp.presentation
 
 import android.os.Bundle
 import androidx.appcompat.app.ActionBar
```

Comment:
the **name again** why Clone? what are you cloning ???

Reply:
The word 'Clone' was removed from the Activity identifiers